recipes/rcorrector/build.sh

```diff
@@ -2,7 +2,9 @@
 
 export C_INCLUDE_PATH=$PREFIX/include
 export OBJC_INCLUDE_PATH=$PREFIX/include
-export CPLUS_INCLUDE_PATH=$PREFIX/include
+export CPLUS_INCLUDE_PATH=$PREFIX/include 
+LD_LIBRARY_PATH=$PATH/lib
```

Comment:
You forgot `export`. You might want `export LD_LIBRARY_PATH=$PATH/lib:$LD_LIBRARY_PATH`. Thankfully conda will end up making a bunch of relative links out of things.

Reply:
see I know what I'm doing ;-)